apps/cookbook/src/app/showcase/item-showcase/item-showcase.component.html

```diff
@@ -270,6 +270,35 @@ <h2 #disclosure>Disclosure</h2>
   </em>
 </p>
 
+<h4>Rotation animation</h4>
+
+<p>
+  It is possible to trigger an animation that rotates the disclosure icon by passing a boolean flag
+  to
+  <code>[rotateIcon]</code>
+  .
```

Comment:
Consider clarifying:
```suggestion
  . When set to <code>true</code> it will rotate the icon 180 degrees and when set to <code>false</code> it will rotate the icon back to it's original position.
```